nullaway/src/main/java/com/uber/nullaway/dataflow/DataFlow.java

```diff
@@ -114,7 +114,7 @@ public ControlFlowGraph load(CfgParams key) {
                             codePath.getLeaf(), (ClassTree) codePath.getParentPath().getLeaf());
                     bodyPath = codePath;
                   }
-                  return CFGBuilder.build(bodyPath, ast, false, false, env);
+                  return CFGBuilder.build(bodyPath, ast, true, false, env);
```

Comment:
I thought we agreed that there would be a command-line flag controlling whether we treat assertions as being on or not?